src/main/java/com/microsoft/azure/functions/worker/broker/JavaFunctionBroker.java

```diff
@@ -96,6 +97,11 @@ private void createInvocationChainFactory(FunctionDefinition functionDefinition,
 					this.sdkParameterAnalyzer.getRegistry()));
 		}
 
+		boolean otelEnabled = Boolean.parseBoolean(System.getenv("JAVA_ENABLE_OTEL"));
```

Comment:
There could be a scenario when System.getenv returns null value and Boolean.parseBoolean with throw error if null value is passed.

https://docs.oracle.com/javase/8/docs//api/java/lang/Boolean.html#parseBoolean-java.lang.String-